### ISSUE TITLE
MangaPark: disable Hentai filter

### ISSUE THREAD

**Issue by zinklog2** [keiyoushi/tachiyomi-extensions-source#3023]
### Source information

MangaPark 1.4.19

### Source language

Multi

### Steps to reproduce

1. search for any series with tag 'hentai'

### Expected behavior

many results should be displayed as the site has a lot of nsfw content

### Actual behavior

you get no results

### Mihon/Tachiyomi version

TachiyomiSY 1.10.5

### Android version

Android 13

### Other details

![image](https://github.com/keiyoushi/extensions-source/assets/110323644/9047d507-c28d-448f-b153-a809ec56ca4b)
this is the default setting for any user, its easy to uncheck on site but no setting to uncheck it on extension. I dont think there even needs to be a toggle just display everything by default.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by AwkwardPeak7** [comment]
the extension does disable it by default

![image](https://github.com/keiyoushi/extensions-source/assets/48650614/596256d2-ef00-4251-9fbb-6cb414a5d1ff)
it enables this option

**Comment by MajSpork** [comment]
I was able to fix the issue by going into webview and logging in to a MangaPark account then going to settings and manually unblocking the tag
